packages/api/src/pre-start/envCheck.ts

```diff
@@ -0,0 +1,17 @@
+import jsonData from './env-required.json';
+
+const envCheck = (): void => {
```

Comment:
Let's make this function a bit more generic so it can be used by both the discord bot & api.

Can you modify `envCheck()` to take 'jsonData' as a parameter?

Reply:
@mattyg do you think we should leave this file in the same place ( packages/api )?

Reply:
yeah for now -- I think we'll want to pull out all the "shared" stuff into a new workspace at some point but we can kick that down the road.